packages/zmarkdown/__tests__/new-html-suite.test.js

```diff
@@ -137,7 +137,7 @@ describe('tables', () => {
       \`ici ok\`| \`gauche | droite\`
     `
 
-    return expect(renderString(input)).resolves.toContain('<td><code>gauche | droite</code></td>')
+    return expect(renderString(input)).resolves.toContain('<td>`gauche</td>')
```

Comment:
I don't think that's correct, the last cell should contain `<code>gauche | droite</code>`:

```md
Titre 1 | Titre 2
--------|--------
`ici ok`| `gauche | droite`
```

Reply:
I thought the same, so I opened an issue on [remark](https://github.com/remarkjs/remark/issues/465), and this is actually the expected behaviour.